ignite/engine/events.py

```diff
@@ -186,9 +212,7 @@ def __init__(self, **kwargs):
             if not hasattr(self, value):
                 setattr(self, value, 0)
 
-    def get_event_attrib_value(self, event_name: Union[EventWithFilter, CallableEvents, Enum]) -> int:
-        if isinstance(event_name, EventWithFilter):
```

Comment:
Here I do not understand why we do not need such checking anymore ? Probably, I do not completely understand your modifications...

Reply:
Because currently all events must be of this type. So a type check is redundant and was thus removed